attendance-list.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
please confirm, but I remember reading that it's recommended to use lowercase i.e. `"utf-8"`